templates/account/signup.html

```diff
@@ -1,96 +1,148 @@
 {% extends "account/base_entrance.html" %}
 {% load allauth i18n %}
 {% load django_bootstrap5 %}
+{% load static %}
+
 {% block head_title %}
-    {% trans "Signup" %}
+  {% trans "Signup" %}
 {% endblock head_title %}
+
 {% block content %}
-    {% element h1 %}
-    {% trans "Sign Up" %}
-{% endelement %}
-{% setvar link %}
-<a href="{{ login_url }}">
-{% endsetvar %}
-{% setvar end_link %}
-</a>
-{% endsetvar %}
-{% element p %}
-{% blocktranslate %}Already have an account? Then please {{ link }}sign in{{ end_link }}.{% endblocktranslate %}
-{% endelement %}
-{% if not SOCIALACCOUNT_ONLY %}
-    {% url 'account_signup' as action_url %}
-    <form method="post" action="{{ action_url }}" class="entrance signup">
-        {% csrf_token %}
-        <h3 class="mb-3">
-            Personal Information
-        </h3>
-        {% bootstrap_field form.username %}
-        {% bootstrap_field form.email %}
-        {% bootstrap_field form.first_name %}
-        {% bootstrap_field form.last_name %}
-        <h3 class="mb-3">
-            Password
-        </h3>
-        {% bootstrap_field form.password1 %}
-        {% bootstrap_field form.password2 %}
-        <h3 class="mb-3">
-            Agreements
-        </h3>
-        <div class="card mb-3">
-            <div class="card-body">
-                <p>
-                    By creating an account, you are agreeing to our terms and conditions.
-                </p>
-                <div class="form-check mb-3">
-                    {{ form.coc_agreement }}
-                    <label class="form-check-label" for="{{ form.coc_agreement.id_for_label }}">
-                        I agree to the <a href="https://2025.conference.pyladies.com/en/coc/"
-    target="_blank"
-    rel="noopener noreferrer">Code of Conduct</a>
-                    </label>
-                    {% if form.coc_agreement.errors %}
+<link rel="stylesheet" href="{% static 'css/custom-login.css' %}">
```

Comment:
Could this be added to the `extra_head` block instead of in the `content` block?